README.md

```diff
@@ -114,12 +151,17 @@ To start or stop individual daemons, the sysvinit script can be used:
 Running unit tests
 ==================
 
-To run build and run all tests, use ctest:
+To run build and run all tests run in `make check`, use ctest:
 
 	cd build
 	make
 	ctest -j$(nproc)
 
+(Note: many targets built from src/test are not run using ctest. 
+Targets starting with "unittest" are run in `make check` and thus can
+be run with ctest. Targets starting with "ceph_test" can not, and should
+be run by hand.)
```

Comment:
could you add a note that `ctest` does not build the dependencies needed to run the test in question?